thing.py

```diff
@@ -0,0 +1,6 @@
+import sys
+import os
```

Comment:
F401 'os' imported but unused
I100 Imports statements are in the wrong order. import os should be before import sys